README.md

```diff
@@ -1,7 +1,7 @@
 # F# |> BABEL
 
 ### The compiler that emits JavaScript you can be proud of. [Try it out!](http://fsprojects.github.io/Fable/repl.html)
-[![Build status](https://ci.appveyor.com/api/projects/status/vlmyxg64my74sik5?svg=true)](https://ci.appveyor.com/project/alfonsogarciacaro/fable)
+[![Mono Build Status](https://travis-ci.org/fsprojects/Fable.svg "Mono Build Status")](https://travis-ci.org/fsprojects/Fable) [![.NET Build Status](https://ci.appveyor.com/api/projects/status/vlmyxg64my74sik5?svg=true ".NET Build Status")](https://ci.appveyor.com/project/alfonsogarciacaro/fable)
```

Comment:
@alfonsogarciacaro I think the diff just visualize it wrong. I tried to merge manually and I still have both badges.

![capture](https://cloud.githubusercontent.com/assets/436605/13423384/decc8dd8-df9a-11e5-9344-9de37fdde5b5.PNG)